src/sdk/sql_cluster_router.cc

```diff
@@ -2321,11 +2321,6 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(const std
                 return {};
             }
 
-            // TODO(denglong): Should support table name with database name
-            if (db.empty()) {
-                *status = {::hybridse::common::StatusCode::kCmdError, "Please use database first"};
-                return {};
-            }
             if (!ExecuteInsert(db, sql, status)) {
```

Comment:
Plz make a comment for `ExecuteInsert`, arg1 `db` can be empty, just a default db. If sql contains db info, this one won't be used.